src/Exception/MissingDependencyException.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
- * @copyright Copyright (c) 2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @copyright Copyright (c) 2016-2017 Zend Technologies USA Inc. (http://www.zend.com)
```

Comment:
File is not changed, year shouldn't be then changed here. I hope we can do some auto checks on it, or at least some scripts to update it automatically - maybe on merge?